misc/python/materialize/buildkite.py

```diff
@@ -95,3 +96,15 @@ def find_modified_lines() -> set[tuple[str, int]]:
                 assert file_path
                 modified_lines.add((file_path, line_nr))
     return modified_lines
+
+
+def upload_artifact(path: Path | str, cwd: Path | None):
```

Comment:
there are a few occurrences of this in the code already, you could make them call your new function too:
```
ci/mkpipeline.py:        spawn.runv(["buildkite-agent", "artifact", "upload", "junit_dummy.xml"])
misc/python/materialize/ci_util/trim_pipeline.py:    spawn.runv(["buildkite-agent", "artifact", "upload", "junit_dummy.xml"])
```

Reply:
👍 

Addressed in https://github.com/MaterializeInc/materialize/pull/22853/commits/d588f7aa03ae51d062a46fa6b21532e25f8e5775.